nixos/modules/services/mail/rspamd.nix

```diff
@@ -58,7 +59,7 @@ let
       };
       type = mkOption {
         type = types.nullOr (types.enum [
-          "normal" "controller" "fuzzy_storage" "proxy" "lua"
```

Comment:
What's an easy fix for this on the nixpkgs side? I can't expect nixos-mailserver to fix this because they need to support multiple versions.

Reply:
I am working on a fix and warning for this

Reply:
Thanks!

I figured out that I can temporarily just use

```nix
{
  services.rspamd.workers.${worker}.type = mkForce "rspamd_proxy";
}
```

in my NixOS config to fix it.

Reply:
@grahamc I haven't changed release-notes before so can you guide me a little?

If I add back support for `type` having the value `proxy` but also then show a warning should I then add my release notes description in the `Backward Incompatibilities` section or the `Other Notable Changes` section?

Reply:
@griff Since after the fix a configuration with the old value still works with the new nixpkgs, it's still compatible. Pretty sure it doesn't need a release note.

Reply:
I don't think reallowing `proxy` is necessary or a good thing to do. These kind of workarounds just increase the complexity of NixOS modules. Nixos-mailserver can fix this with https://gitlab.com/simple-nixos-mailserver/nixos-mailserver/merge_requests/142.